### PR TITLE
fixbug: there's a problem about using external LLM of huoshan #1429

### PR DESCRIPTION
**Pull request by iorisa**
**Features**
- #1429

### CODE REVIEW

File: metagpt/provider/ark_api.py [hunk 1]
```diff
@@ -16,11 +37,45 @@ class ArkLLM(OpenAILLM):
     见：https://www.volcengine.com/docs/82379/1263482
     """
 
+    aclient: Optional[AsyncArk] = None
```
Comment (better629) [inline metagpt/provider/ark_api.py:40]: Seems ark has a openai compatible interface, should we use the official sdk?

Comment (better629) [reply metagpt/provider/ark_api.py:40]: ok, to support also ak/sk

File: metagpt/utils/token_counter.py [hunk 2]
```diff
@@ -187,6 +187,14 @@
     "mixtral-8x7b": {"prompt": 0.4, "completion": 1.6},
 }
 
+# https://console.volcengine.com/ark/region:ark+cn-beijing/model
+DOUBAO_TOKEN_COSTS = {
```
Comment (better629) [inline metagpt/utils/token_counter.py:191]: There also doubao-xx related token cost under `TOKEN_COSTS`, you can delete it.

Comment (iorisa) [reply metagpt/utils/token_counter.py:192]: 截屏2024-08-07 15 08 09

The model name has the version number embedded in it.
`DOUBAO_TOKEN_COSTS` provides model names without version numbers for `pricing_plan` configuration.

File: metagpt/provider/ark_api.py (hunk 1 above)
Comment (iorisa) [reply metagpt/provider/ark_api.py:40]: The openai compatible interface is not working, so it's replaced by sdk.

**Comment by better629** [comment]
lgtm

**Pull request merged**